core/src/main/java/kafka/server/builders/ReplicaManagerBuilder.java

```diff
@@ -179,6 +180,7 @@ public ReplicaManager build() {
         if (metadataCache == null) throw new RuntimeException("You must set metadataCache");
         if (logDirFailureChannel == null) throw new RuntimeException("You must set logDirFailureChannel");
         if (alterPartitionManager == null) throw new RuntimeException("You must set alterIsrManager");
+        if (brokerTopicStats == null) brokerTopicStats = new BrokerTopicStats(Option.apply(config));
```

Comment:
nit: Instead of `Option`, use java `Optional`

Reply:
Since we're moving to java now, it makes sense to use `optional`. PR updated.